custom_components/plugwise-beta/sensor.py

```diff
@@ -275,7 +275,7 @@ def __init__(self, api, coordinator, name, dev_id, sensor, sensor_type):
             self._name = f"Smile {self._name}"
             self._via_id = None
 
-        self._unique_id = f"{dev_id}-{sensor}"
+        self._unique_id = f"{dev_id}-{sensor_type}"
```

Comment:
Sensor_type will not work, will lead to duplicate unique_ids. Use sensor instead.
There can be equal sensors but then the devid will be different.

Reply:
c4090b8 reverts it